src/core/quadtree.js

```diff
@@ -0,0 +1,462 @@
+/*jslint vars: true, nomen: true, plusplus: true, continue:true, forin:true */
+/*global Node, QNode */
+
+/*
+
+  Copyright (c) 2014 Michael Sander (speedplane)
+  Based loosely on code written by (c) 2011 Mike Chambers.
```

Comment:
Actually it looks like we're using Apache, not the MDL, so I'll switch it to that.